runhouse/docker/slim/Dockerfile

```diff
@@ -43,7 +48,7 @@ RUN echo "[supervisord]" > /etc/supervisor/conf.d/supervisord.conf && \
     echo "stdout_logfile=/var/log/sshd.log" >> /etc/supervisor/conf.d/supervisord.conf && \
     echo "stderr_logfile=/var/log/sshd.err" >> /etc/supervisor/conf.d/supervisord.conf && \
     echo "[program:runhouse]" >> /etc/supervisor/conf.d/supervisord.conf && \
```

Comment:
Should we document that we explicitly run the runhouse restart command in the foreground (--no-screen)? And why we use `restart` instead of `start`?